src/OpenLoco/src/Input/MouseInput.cpp

```diff
@@ -467,7 +467,8 @@ namespace OpenLoco::Input
                         auto tool = WindowManager::find(ToolManager::getToolWindowType(), ToolManager::getToolWindowNumber());
                         if (tool != nullptr)
                         {
-                            tool->callToolDragContinue(ToolManager::getToolWidgetIndex(), x, y);
+                            // TODO: Handle widget id properly for tools.
+                            tool->callToolDragContinue(ToolManager::getToolWidgetIndex(), WidgetId::none, x, y);
```

Comment:
I'm not sure what the point of the add parameter is here. Don't all tools have their widget id set, anyway? (Indeed that's what `getToolWidgetIndex()` is)

Reply:
The thing is that index and id are two different things, the tools just abuse the type to have a unique identifier, that's what the todo is for.

Reply:
There is an open PR that refactors tools so that it doesn't have to pass the tool index or widget https://github.com/OpenLoco/OpenLoco/pull/2905